contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -752,13 +752,39 @@ TsqlExpressionContains(char *colId, Node *search_expr, core_yyscan_t yyscanner)
 static Node *
 makeToTSVectorFuncCall(char *colId, core_yyscan_t yyscanner, Node *pgconfig)
 {
-    Node *col;
-    List *args;
-    Node *replaceSpecialCharsFunc;
-    List *replaceSpecialCharsArgs;
-
-    /* Create a ColumnRef node for the column */
-    col = makeColumnRef(colId, NIL, -1, yyscanner);
+    Node	*col;
```

Comment:
Have we considered column alias as well?
did we try various cases where . can be part of a column reference? say schema_name.table_name.col_name
did we consider cases where we use " or ' quotes around the reference?

Reply:
Column alias is not allowed inside CONTAINS clause so we are good with that

Reply:
Column aliases are not allowed in `CONTAINS` predicate, and throws error.